examples/strings/strings.bal

```diff
@@ -64,18 +64,20 @@ public function main() {
     boolean contains = statement.includes("Town", 7);
     io:println("Contains: ", contains);
 
-    // Format a `string` according to the given format arguments.
+    // Format a `string` according to the given format arguments
+    // using string templates.
     string name = "Sam";
-    int marks = 90;
+    int mrk = 90;
     string[] subjects = ["English", "Science"];
-    float average = 71.5;
-    string s8 = io:sprintf("%s scored %d for %s and has an average of %.2f.",
-     name, marks, subjects[0], average);
-    io:println("Sprintf: ", s8);
+    float avg = 71.5;
+    string s8 = string `${name} scored ${mrk} for ${subjects[0]} and has an
+                  average of ${avg}`;
```

Comment:
Shall we adjust here in order to get 80 characters per line?

Reply:
It is within 80 characters.